src/GovUk.Education.ExploreEducationStatistics.Content.Model/Extensions/ReleaseExtensions.cs

```diff
@@ -1,10 +1,26 @@
 ﻿using System;
 using System.Linq;
+using static GovUk.Education.ExploreEducationStatistics.Common.Services.FileStoragePathUtils;
 
 namespace GovUk.Education.ExploreEducationStatistics.Content.Model.Extensions
 {
     public static class ReleaseExtensions
     {
+        /// <summary>
+        /// The storage blob path of the "All files" zip file on a Release.
+        /// </summary>
+        /// <param name="release"></param>
+        /// <returns></returns>
+        public static string AllFilesZipPath(this Release release)
```

Comment:
Wonder if it would be better to create an 'All file' `File` in the database when we publish the release to avoid needing these kinds of paths. One for another ticket?

Reply:
Yes it's a major pain. I've written up https://dfedigital.atlassian.net/browse/EES-1938 on all the problems it's causing.